README.md

```diff
@@ -68,6 +73,12 @@ To add requirements used by the library, update `dependencies` in `pyproject.tom
 - For test dependencies, update `dependencies` under `[tool.hatch.envs.test]`.
 
 ### Useful Hatch commands
+Spawn a shell within an environment for development:
```

Comment:
Could we move this further down in this block? I think other Hatch commands are more useful -- especially the ones to run lint and tests.